zhaquirks/tuya/ts0601_sensor.py

```diff
@@ -71,3 +76,25 @@
     .skip_configuration()
     .add_to_registry()
 )
+
+(
+    TuyaQuirkBuilder("_TZE200_pay2byax", "TS0601")  # Cusam ZG-102ZL
+    .applies_to("_TZE200_n8dljorx", "TS0601")
+    .tuya_sensor(
+        dp_id=101,
+        attribute_name="measured_value",
+        type=t.uint16_t,
+        fallback_name="Illuminance",
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Need `translation_key`

Reply:
Translation key is optional if a device class is set.  HA already has translations and icons for illiminance and it is working perfectly fine.
![image](https://github.com/user-attachments/assets/83f8a5c9-5447-417f-8510-d4e6d1a60faf)